db/collab.go

```diff
@@ -0,0 +1,61 @@
+package db
+
+import (
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/labstack/echo/v4"
+	"github.com/uclaacm/teach-la-go-backend/httpext"
+	"golang.org/x/net/websocket"
+)
+
+// Session describes a collaborative coding environment.
+type Session struct {
+	// Map UIDs to their websocket.Conn
+	Conns map[string]*websocket.Conn
+}
+
+// Maps session IDs to Session object
+var sessions map[string]Session
+var sessionsLock sync.Mutex
```

Comment:
Should be replaced with tabs but here's what it's complaining about.

```suggestion
// Maps session IDs to Session object
var (
    sessions        map[string]Session
    sessionsLock sync.Mutex
)
```